sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsView.java

```diff
@@ -411,6 +414,7 @@ public HorizontalLayout createStatusFilterBar() {
 				relevanceStatusFilter.setId("relevanceStatusFilter");
 				relevanceStatusFilter.setWidth(140, Unit.PERCENTAGE);
 				relevanceStatusFilter.setNullSelectionAllowed(false);
+				relevanceStatusFilter.setTextInputAllowed(false);
```

Comment:
What's the reason for this change?

Reply:
I stumbled upon the fact that you could currently write text in the relevance filter, which doesn't make any sense as the typed-in-text wasn't processed anyway. So I thought I'd quickly disable it while working in the same file anyway, for a more cleaner UI.